MkDocs/hpu_8.6/prereqs.md

```diff
@@ -25,14 +25,14 @@ o Click `Create an IBM ID` link
 1. From the grid view...
 2. To add the saved view...click on the `Filter` icon, click on `Edit` next to `Query`, and choose `EUDEMO_ST`. Click `Ok` and `Apply`. 
 
-    ![img](/img/hpu_8.6/prereq1.1.png){: style="height:172px;width:200px;margin-left:40px"}
-    ![img](/img/hpu_8.6/prereq1.2.png){: style="height:172px;width:200px;margin-left:40px"}
-    ![img](/img/hpu_8.6/prereq2.1.png){: style="height:172px;width:200px;margin-left:40px"}
+    ![img](/img/hpu_8.6/prereq1.1.png)
```

Comment:
You reference these image files prereq1.1.png  and others but don't include them in your pull request.  You are required to make sure that your pull request is functionally complete without files missing or having errors.  Please add the missing image files to this PR.   Additionally you should first discuss the topics/lab you contributing before you go to far to ensure someone else isn't already working on such a lab and that the content makes sense for self paced or instructor led labs.